tests/ecosystem/upgrade/conftest.py

```diff
@@ -0,0 +1,309 @@
+import logging
+
+from ocs_ci.ocs import constants, resources
+import pytest
+import threading
```

Comment:
Pep8 imports. So threading in first category, pytst in second,  imports from ocs in third category and empty line between them.
```
Standard library imports.

Related third party imports.

Local application/library specific imports.
```

Reply:
Done